### PR TITLE
JavaScript error occurred in main process on macOS close window keys combination #1020

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

**Comment by sudhirverma** [comment]
test this

**Comment by dgolovin** [comment]
test this

**Comment by sudhirverma** [comment]
test this

File: main/index.js [hunk 1]
```diff
@@ -49,7 +49,10 @@ app.on('quit', function(event, exitCode) {
 // initialization and is ready to create browser windows.
 app.on('ready', function() {
   globalShortcut.register('CmdOrCtrl+W',()=>{
```
Comment (dgolovin) [inline main/index.js]: Is there should be a code that removes global shortcut when installer app is closed? What happens if you run installer with this fix, then exit it and then try to use Cmd+W in any other window?

**Pull request merged**